submissions/benjaminKamena/phpDataase/ordersQueryDb.php

```diff
@@ -0,0 +1,40 @@
+<?php
+include('connect.php');
+
+if(isset($_GET['customer_id'])){
```

Comment:
So, this is what you had previously 
![image](https://user-images.githubusercontent.com/43061141/106254436-fd19dd80-6218-11eb-9e75-bf32966f7720.png)

How is it possible for you to have a GET request that contains `customer_id`. Remember the question said then click on the `View` button, it should show is this page.